e2e/client/playwright/monitoring.duplication.spec.ts

```diff
@@ -31,11 +31,11 @@ test('duplicate to personal space', async ({page}) => {
     await restoreDatabaseSnapshot();
 
     await page.goto('/#/workspace/personal');
-    await expect(page.locator(s('list-item-placeholder'))).not.toBeVisible();
 
-    await expect(page.locator(articleSelector)).toHaveCount(0);
+    await expect(page.locator(s('list-item-placeholder'))).toBeVisible();
+    await expect(page.locator(s('list-item-placeholder'))).not.toBeVisible();
 
-    const itemCountInPersonalSpace = await page.locator(s('article-item')).count();
+    const itemCountInPersonalSpace = await page.locator(s('article-item')).count(); // need to be 1
```

Comment:
> // need to be 1

It does not need to be 1. It will be one now, but if I update database dump it might as well be 2 or 3 or 4. That's why the test is not testing if could goes from 1 to 2, but is testing if it increased by 1.